content/nginxaas-azure/quickstart/security-controls/oidc.md

```diff
@@ -12,16 +12,227 @@ type:
 
 Learn how to configure F5 NGINXaaS for Azure with OpenID Connect (OIDC) authentication.
 
+There are currently two methods available for setting up OIDC authentication.
+
+1. Using Native OIDC implementation (Introduced from NGINX Plus R35)
+   
+   This method applies to NGINX Plus Release 35 and later. In earlier versions, NGINX Plus relied on an njs-based solution, which required NGINX JavaScript files, key-value stores, and advanced OpenID Connect logic. In the latest NGINX Plus version, the new [OpenID Connect module](https://nginx.org/en/docs/http/ngx_http_oidc_module.html) simplifies this process to just a few directives.
```

Comment:
This paragraph starts with "this method" but is followed by prerequisites that apply to both methods.

Maybe it's better to move this paragraph to Line 33 after the Native OIDC heading.